components/odh-notebook-controller/run-e2e-test.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+TEST_NAMESPACE="odh-notebook-controller-system"
+
+# setup and deploy the controller
+oc new-project $TEST_NAMESPACE -n $TEST_NAMESPACE --skip-config-write
+IFS=':' read -r -a CTRL_IMG <<< "${ODH_NOTEBOOK_CONTROLLER_IMAGE}"
+export IMG="${CTRL_IMG[0]}"
+export TAG="${CTRL_IMG[1]}"
+make deploy -e K8S_NAMESPACE=$TEST_NAMESPACE
+
```

Comment:
Given that we are also build`KF_IMG` in openshift-ci, can you also update this to use [those](https://github.com/opendatahub-io/kubeflow/blob/v1.6-branch/components/odh-notebook-controller/Makefile#L6) images.

The exposed env variable for KFNBC is `KF_NOTEBOOK_CONTROLLER` image